mod/01_rating/set/type/metric_value.rb

```diff
@@ -106,31 +106,40 @@ def create_source
 end
 
 format :html do
-  def default_new_args args
+  def get_structure args
```

Comment:
What structure will be used depends on the context. If a new metric value from is with company in the argument, a metric value form without a company field will be shown and so on.

We did not make a correct structure for a default new metric value form so I created a new structure with all fields. It will be shown when there is no company/source/metric in the argument.